xapi-xenopsd.opam

```diff
@@ -1,29 +1,20 @@
 opam-version: "1.2"
+name: xapi-xenopsd
 maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://github.com/xapi-project/xenopsd"
-build: [
-	["./configure"]
-	[make]
-]
-install: [
- 	["./setup.bin" "-install"]
-]
-remove: [
-        ["make" "setup.data"]
-        ["./setup.bin" "-uninstall"]
-        ["ocamlfind" "remove" "xenopsd"]
-]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
 depends: [
-  "oasis" {build}
-  "ocamlfind" {build}
+  "jbuilder" {build}
+  "base"
```

Comment:
Do we need this dependency?

Reply:
No, I tried using it, but I ran into conflicting dependecies. I ended up writing a work around and forgot to remove this.